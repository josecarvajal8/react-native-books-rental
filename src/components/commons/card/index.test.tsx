import React from 'react';
import { Text } from 'react-native-animatable';
import renderer from 'react-test-renderer';
import Card from '.';
const mockProps = {
    children: () => <Text>{'hello'}</Text>,
    action: () => null
}
test('Render card component correctly', () => {
    const cardCmp = renderer.create(<Card {...mockProps} />).toJSON();
    expect(cardCmp).toMatchSnapshot();
});