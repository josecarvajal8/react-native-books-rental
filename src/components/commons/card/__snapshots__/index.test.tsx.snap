// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render card component correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#ffffff",
      "borderRadius": 14.492753623188406,
      "height": 163.77232142857144,
      "opacity": 1,
      "shadowColor": "rgba(46, 48, 48, 1)",
      "shadowOffset": Object {
        "height": 1.4888392857142858,
        "width": 1.8115942028985508,
      },
      "shadowOpacity": 0.3,
      "shadowRadius": 2,
      "width": 688.4057971014493,
    }
  }
/>
`;
